puppet/manifests/development.pp

```diff
@@ -55,14 +55,20 @@
 	admin_user        => $config[admin][user],
 	admin_email       => $config[admin][email],
 	admin_password    => $config[admin][password],
-	plugins           => $config[plugins],
-	themes            => $config[themes],
 
 	extensions        => $extensions,
 
-	require  => [
+	require => [
 		Class['chassis::php'],
 		Package['git-core'],
 		Class['mysql::server'],
 	]
 }
+
+# Tasks wp::plugin and wp::theme run onlyif "/usr/bin/wp core is-installed":
+# sequence chassis::content after chassis::wp.
+-> chassis::content { $config['hosts'][0]:
```

Comment:
It's possible that there's a way to use `require` in place of `->` here, which might be conceptually preferable, but this works and I burned too much time trying to understand puppet defs _vs_ classes to continue pursuing that approach. :)

Reply:
We generally only use `require`. For this, it will be:
```
require => Chassis::Wp[ $config['hosts'][0] ],
```

Reply:
btw, the rule for referencing is:
* Take the declaration line which looks like `foo::bar { "quux":`
* Split the first bit (the *resource type*) by namespace separator (`::`)
* Uppercase the first letter of each piece
* Combine together with namespace separator
* Add the second bit (the *resource name*) in square brackets

So, `class { 'foo':` -> `Class['foo']`, `chassis::wp{ $x:` -> `Chassis::Wp[ $x ]`